tst/EnergyPlus/unit/Fixtures/HVACFixture.hh

```diff
@@ -6,6 +6,24 @@
 
 // EnergyPlus Headers
 #include "EnergyPlusFixture.hh"
+#include <EnergyPlus/CurveManager.hh>
```

Comment:
@EnergyArchmage Since you moved all the clear_state() methods to the EnergyPlusFixture TearDown(), you can (and should) get rid of these includes here. You might need to put includes into specific unit test files again but that will show up when you go to compile.